changelog: images update part2 (#210)
CleverCloud/documentation: static/partials/language-specific-deploy/dotnet.md

### .NET version

> The default version used on Clever Cloud is `8.0`. You can change it by setting the `CC_DOTNET_VERSION` environment variable to `6.0`. No support will be provided for non-LTS and older versions.

Reviewer: ```suggestion
The default version used on Clever Cloud is `8.0`. You can change it by setting the `CC_DOTNET_VERSION` environment variable to `6.0`. Clever Cloud doesn't provide support for non-LTS and older versions.
```

First, you need to add the `APP_FOLDER=SimpleFeedReader` environment variable to define the application folder inside the Git repository.

> During deployment, the `SimpleFeedReader.csproj` file and the target framework `net8.0` are automatically detected. Then, the .NET project is published:

Reviewer: ```suggestion
During deployment, the runtime automatically detects the `SimpleFeedReader.csproj` file and the target framework `net8.0`. Then, it publishes the .NET project:
```
